src/components/custom/dashboard/trainer/pages/newclients/components/ClientDetailsModal.jsx

```
@@ -0,0 +1,550 @@
+                      clientDetails.client.clientProfile.profileImage,
+                      `${clientDetails.client.clientProfile.firstName} ${clientDetails.client.clientProfile.lastName}`
+                    );
+                  }
+                }}
+                onClick={() =>
+                  onProfileImageClick(
+                    clientDetails.client.clientProfile.profileImage,
+                    `${clientDetails.client.clientProfile.firstName} ${clientDetails.client.clientProfile.lastName}`
+                  )
+                }
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)